server/src/controllers/service-transaction.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from "express";
+import AboutServiceTransactionProvider from "../providers/about-service-transactions";
```

Comment:
what does the name about-service-transactions has to do with claim?

Reply:
I will rename about-claim-transactions